Packs/QualysVulnerabilityManagement/Integrations/QualysVulnerabilityManagement/QualysVulnerabilityManagement.py

```diff
@@ -0,0 +1,1499 @@
+            'recipient_group', 'ips', 'asset_group_ids', 'policy_id', 'host_id', 'instance_string'
+        ],
+    },
+    'qualys-virtual-host-manage': {
+        'args': ['action', 'ip', 'network_id', 'port', 'fqdn'],
+    },
+    'test-module': {
+        'args': []
+    },
+}
+
+# Dictionary for arguments used by Qualys API
+args_values: Dict[str, Any] = {}
+
+# Dictionary for arguments used internally by this integration
+inner_args_values: Dict[str, Any] = {}
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+
+    def __init__(self, base_url, username, password, verify=True, proxy=False,
+                 ok_codes=tuple(), headers=None):
+        super().__init__(base_url, verify, proxy, ok_codes, headers, auth=(username, password))
+
+    def command_http_request(self, command_api_data: Dict[str, str]) -> Union[str, bytes]:
+        """
+        Make a http request to Qualys API
+        Args:
+            command_api_data: Information about the API request of the requested command
+        Returns:
+            response from Qualys API
+        Raises:
+            DemistoException: can be raised by the _http_request function
+        """
+        return self._http_request(
+            method=command_api_data['call_method'],
+            url_suffix=command_api_data['api_route'],
+            params=args_values,
+            resp_type=command_api_data['resp_type'],
+            timeout=60
+        )
+
+
+''' HELPER FUNCTIONS '''
+
+
+def json_get_path(json_obj: Dict[str, Any], path_str: str) -> Dict[str, Any]:
+    """
+    This function takes a path and gets the object at the specified path in the dictionary
+    Args:
+        json_obj: Dictionary
+        path_str: String of nested keys '.' separated, making a path
+    Returns:
+            Dict: dictionary nested inside the original dictionary or the original dict
+    Raises:
+        KeyError: raised if a bad or incorrect path was provided or bad json object
+        TypeError: raised if json_obj is not a dictionary
+    """
+    if len(path_str) == 0:
+        return json_obj
+    path_keys_list_path = path_str.split(".")
+    current_json_obj = json_obj
+    for key in path_keys_list_path:
+        current_json_obj = current_json_obj[key]
+    return current_json_obj
+
+
+def create_ip_list_dict(res_json: Dict[str, Any], type_of_dict: str) -> Dict[str, Any]:
+    """
+    Creates a dictionary of a range type of ips or single address type of ips
+    Args:
+        res_json: Dictionary received from ip list command with 'Address' or 'Range' keys
+        type_of_dict: The wanted type of dictionary: 'Range' or 'Address'
+    Returns:
+        A dictionary with the specified type of ips values
+    """
+    ips_dict = {}
+
+    if type_of_dict in res_json:
+        ips = res_json[type_of_dict]
+        # In case a single value returned it can be either a Dict or a str
+        if isinstance(ips, str) or isinstance(ips, Dict):
+            ips_dict = {'0': ips}
```

Comment:
Actually no, I added this just for the case when there's a single value. I made it in the above format just so it will be consistent with the format returned from Qualys when there's more than a single value.